domains/DataCollection/Forms/IdentificationForm/index.js

```diff
@@ -57,7 +57,15 @@ const IdentificationForm = ({
 
             formObject.dob = `${values.Month || '00'}/${values.Day || '00'}/${values.Year || '0000'}`;
 
-            formObject.searchIndex = `${values.fname || ''} ${values.lname || ''}`;
+            formObject.searchIndex = [
+              values.fname,
+              values.lname,
+              values.communityname
+            ].map((result)=>{
+              if(result){
+                return result.toLowerCase().trim()
```

Comment:
I'm a bit confused for what should be in the searchIndex. When looking at b4a production I see this: [
  "hdwdszvuzd",
  "miguel angel ",
  "cherri",
  "chimbara",
  "batey la barza"
]

which includes 5 keys, objectId (not positive), fname, nickname (?), lname, community. 2 of these values regardless of which way you cut it are missing. Are they not required?

Reply:
@jamccomb92 yes! I forgot nickname so thank you for that

So here's the discrepancy

- I realized that no one would ever be searching by objectId, so I excluded that from arrays moving forward. re-running the batch job I created to update the entire model is *really* time-consuming
- I felt that fname, lname, and community would suffice and then got pinged that nickname would be nice to add so then started adding these, but forgot to put these in this string array so thank you